.changeset/nervous-starfishes-report.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-home': patch
```

Comment:
you'll need a changeset for plugin-search as well right?

Reply:
there is just a storybook addition to the search plugin now so I think we don't need it!

Reply:
Yup!  Now that the change is purely storybook-related, none needed for search.